src/Operators/finitedifference.jl

```diff
@@ -1728,9 +1728,8 @@ stencil_interior_width(::FCTZalesak, A_space, Φ_space, Φᵗᵈ_space) =
     )
 
 
-    dxⱼ = Geometry.LocalGeometry(center_space, idx - half, hidx).∂x∂ξ[1]
-    dxⱼ₊₁ = Geometry.LocalGeometry(center_space, idx + half, hidx).∂x∂ξ[1]
-
+    dxⱼ = Geometry.LocalGeometry(center_space, idx - half, hidx).∂x∂ξ[3, 3]
+    dxⱼ₊₁ = Geometry.LocalGeometry(center_space, idx + half, hidx).∂x∂ξ[3, 3]
```

Comment:
@simonbyrne what is a robust way to get `dz` across all types of domains (1D-3D), both uniform and stretched? I had tested the `∂x∂ξ[1]` for both 1D and extruded 3D spherical domains and it seemed to be working as expected

Reply:
#884 was a reminder for ourselves to add unit tests for this indeed

Reply:
Related issue: #884.

Reply:
We discussed removing the dx altogether. Related PR: #907